packages/components/src/components/dropdown-item/dropdown-item.tsx

```diff
@@ -7,6 +7,8 @@ import { Component, Element, Prop } from '@stencil/core';
 export class DropdownItem {
     @Element() el: HTMLDivElement;
     @Prop() selected: boolean = false;
+    @Prop() disabled: boolean = false;
```

Comment:
I know this component is badly documented, but should we be some good "scouts" and start doing it?